client/models/channel.coffee

```diff
@@ -77,7 +77,14 @@ module.exports = class Channel extends Backbone.Model
         'auctions'
       ], association
     else if type is 'partner'
-       false
+      _.contains [
```

Comment:
yeah good point --  i have a feeling these associations will be tweaked often so i'll leave it in for now but definitely true about DRYing up code